packages/solecs/package.json

```diff
@@ -21,32 +21,23 @@
     "release": "npm publish --access=public",
     "test": "forge test"
   },
-  "devDependencies": {
+  "dependencies": {
     "@ethersproject/abi": "^5.7.0",
     "@ethersproject/providers": "^5.7.2",
-    "@solidstate/contracts": "^0.0.52",
-    "@typechain/ethers-v5": "^10.2.0",
-    "@types/mocha": "^9.1.1",
     "ds-test": "https://github.com/dapphub/ds-test.git#c9ce3f25bde29fc5eb9901842bf02850dfd2d084",
     "ethers": "^5.7.2",
+    "memmove": "https://github.com/dk1a/memmove.git#ffd71cd77b1708574ef46a667b23ca3a5cc9fa27",
+    "solmate": "https://github.com/transmissions11/solmate.git#9cf1428245074e39090dceacb0c28b1f684f584c"
+  },
+  "devDependencies": {
+    "@solidstate/contracts": "^0.0.52",
```

Comment:
I think `@solidstate/contracts` should be a dependency so it's automatically installed by consumers

Reply:
oops, good catch